bolt-sidecar/src/state/execution.rs

```diff
@@ -291,6 +295,11 @@ impl<C: StateFetcher> ExecutionState<C> {
         // Check if the max_fee_per_gas would cover the maximum possible basefee.
         let slot_diff = target_slot.saturating_sub(self.slot);
 
+        // Verify max slot diff
+        if slot_diff > 64 {
+            return Err(ValidationError::SlotTooHigh(target_slot, self.slot + 64));
+        }
+
```

Comment:
While I support this change, the fact that we need it is weird.

If you take a look below:
https://github.com/chainbound/bolt/blob/f553103879ce6aec5a7ec1a06d7f14bc9b9c5746/bolt-sidecar/src/state/consensus.rs#L114-L122

you can see that when doing consensus checks we already verify that the slot of the inclusion request is in some bounds, so the a `slot_diff` greater than `64` should not happen because it should be larger then `furthest_slot`.

Can you please double check if there is another hidden error there? Happy to debug it together if needed

Reply:
Ah you're right, I missed this! Gonna remove the check.

Reply:
note that this check might be skipped if the user has set --unsafe-skip-consensus-checks when running the sidecar.